### PR TITLE
Ensure the HTML App test actually has a browser to run on.

### PR DESCRIPTION
**Pull request by l-1squared**

### CODE REVIEW

File: .github/workflows/jgiven_branch_and_pr_build.yml [hunk 1]
```diff
@@ -12,6 +12,8 @@ jobs:
         java: [ '11', '13']
         junit: ['4.12','4.13']
     steps:
+      - name: Ensure firefox is installed
+        run: sudo apt-get install firefox
```
Comment (fudler) [inline .github/workflows/jgiven_branch_and_pr_build.yml:16]: Will the sudo step work or could it crash due to the lack of credentials/permissions?

File: .github/workflows/jgiven_examples_build.yml [hunk 2]
```diff
@@ -22,7 +22,8 @@ jobs:
           && runGradleTestOnGivenProject example-projects/kotlin/build.gradle 1.1-t
       - name: Test JUnit5 Example Project
         run: >
-          source scripts/source_files/helper_functions.sh
+          sudo apt-get install firefox
+          && source scripts/source_files/helper_functions.sh
```
Comment (fudler) [inline .github/workflows/jgiven_examples_build.yml:26]: Same as above.

File: jgiven-tests/src/test/java/com/tngtech/jgiven/report/html5/Html5AppTest.java [hunk 3]
```diff
@@ -4,37 +4,21 @@
...
 import org.junit.runner.RunWith;
-import org.openqa.selenium.Dimension;
 import org.openqa.selenium.OutputType;
 import org.openqa.selenium.TakesScreenshot;
 import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.chrome.ChromeOptions;
+
+import java.io.IOException;
+import java.util.*;
```
Comment (fudler) [inline jgiven-tests/src/test/java/com/tngtech/jgiven/report/html5/Html5AppTest.java:21]: Do we need this huge import?

File: .github/workflows/jgiven_branch_and_pr_build.yml (hunk 1 above)
Comment (l-1squared) [reply .github/workflows/jgiven_branch_and_pr_build.yml:16]: we generally have root and installation permissions on the ci runners. (We also install jeckyll for releases and sbt stuff for our tests).
In fact, it already worked: https://github.com/TNG/JGiven/actions/runs/4465801439/jobs/7843220117

File: jgiven-tests/src/test/java/com/tngtech/jgiven/report/html5/Html5AppTest.java (hunk 3 above)
Comment (l-1squared) [reply jgiven-tests/src/test/java/com/tngtech/jgiven/report/html5/Html5AppTest.java:21]: probably not

File: .github/workflows/jgiven_branch_and_pr_build.yml (hunk 1 above)
Comment (fudler) [reply .github/workflows/jgiven_branch_and_pr_build.yml:16]: Then everything is fine ;)

**Pull request merged**